src/models/line.js

```diff
@@ -121,8 +123,8 @@ nv.models.line = function() {
                         .interpolate(interpolate)
                         .defined(defined)
                         .x(function(d,i) { return nv.utils.NaNtoZero(x0(getX(d,i))) })
-                        .y0(function(d,i) { return nv.utils.NaNtoZero(y0(getY(d,i))) })
-                        .y1(function(d,i) { return y0( y.domain()[0] <= 0 ? y.domain()[1] >= 0 ? 0 : y.domain()[1] : y.domain()[0] ) })
+                        .y0(function(d,i) { return y0(getYLower(d,i)).length == 0 ? nv.utils.NaNtoZero(y(getY(d,i))) :  y0(getYLower(d,i))})
+                        .y1(function(d,i) { return y0(getYUpper(d,i)).length == 0 ? y( y.domain()[0] <= 0 ? y.domain()[1] >= 0 ? 0 : y.domain()[1] : y.domain()[0] ) : y0(getYUpper(d,i))})
```

Comment:
Yes. These codes are correct.
When I use "ylower",  line 126's code choose y0().
And, y1 is same way.